wordpress-mobile/WordPress-iOS
# [Stats Refresh] Remove unused row types

Fixes #n/a

This a bit of code cleanup/refactor to remove row types that are no longer necessary.

With the conversion to two-column stat cards, `SimpleTotalsStatsRow` was no longer used, so that's gone.
That left only Insights _Publicize_ using `SimpleTotalsCell` (via `SimpleTotalsStatsSubtitlesRow`). With a small change, it's now using `TopTotalsCell`, and thus `SimpleTotalsCell` has been removed.

As an added bonus, this fixes an issue where the _Publicize_ subtitles didn't show until you scrolled up and back down. 🎉 for accidental fixes!

To test:
- Go to Insights Publicize.
- Verify it renders correctly. And with subtitles on the first try!

<img width="400" alt="publicize" src="https://user-images.githubusercontent.com/1816888/59957728-7146a980-9458-11e9-877f-6602f43d12f9.png">

Update release notes:
